pennylane/qnode.py

```diff
@@ -363,6 +363,43 @@ def get_best_method(device, interface):
                 except qml.QuantumFunctionError:
                     return qml.gradients.finite_diff, {}, device
 
+    @staticmethod
+    def get_best_method_str(device, interface):

```

Comment:
The existing name is great and follows the previous convention well :+1: We have it on our radar to start moving away from names that start with `get`, `set`, `compute`, etc. just for readability purposes. :slightly_smiling_face: 

Note that the change will affect many changes in this PR (including the changelog entry).
```suggestion
    def best_method_str(device, interface):
```